### DISCUSSION TITLE
How to connect Docker using 'https' ?

### DISCUSSION THREAD

**Discussion by MelonPark-JP** [mrgnr/camus#46] [Q&A]
I want to make video conference on local network.
I pulled docker image of mrgnr/camus and run.
Other machine can easily connect to http:target IP:5000.
However, we cannot connect it by 'https.'
Could you tell me how to connect it by 'https://' ?

**Comment by mrgnr** [comment]
Hi @Masa-Kawa, I'd recommend running Nginx in front of the Camus server and doing SSL termination with Nginx. If running on a local network (i.e. not on the public internet), you'll need to generate your own SSL certificates to use HTTPS.

[This tutorial](https://docs.camus.chat/en/latest/tutorials/rpi-deployment.html) might be helpful for what you're trying to do (it's written for Raspberry Pi but most of the instructions apply to any Debian/Ubuntu-based system).

**Reply by MelonPark-JP** [reply]
Hi Morgan. Thank you for your reply. I'll try to make the system according to your advice. I am a novice for such system. If I could, I want to use pre-build docker container of camus.
